.github/workflows/ci.yml

```
@@ -0,0 +1,100 @@
+        with:
+          name: backend-build-ubuntu
+          path: backend/build
+
+  backend_fedora_container:
+    name: Build Backend (Fedora container)
+    runs-on: ubuntu-latest
+    needs: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run build inside Fedora container
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)